enterprise/j2ee.core/nbproject/org-netbeans-modules-j2ee-core.sig

```diff
@@ -26,6 +26,7 @@ CLSS public java.lang.Object
 cons public init()
 meth protected java.lang.Object clone() throws java.lang.CloneNotSupportedException
 meth protected void finalize() throws java.lang.Throwable
+ anno 0 java.lang.Deprecated(boolean forRemoval=false, java.lang.String since="9")
```

Comment:
can be removed i think

Reply:
Regenerated sig file which removed the line.